frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -88,4 +88,19 @@ export async function writeActivity(tripId, activityId, newValues) {
     console.log(e);
     return false;
   }
+}
+
+/**
+ * Get the value of a reference. 
+ * 
+ * @param {Reference} ref Reference to get the value of.
+ * @param {string} ignoreValue The "null" or "none" value that ref could be.
+ * @param {string} defaultValue Value to return if ref.current.value === ignoreValue.
+ * @return defaultValue if ref.current.value === ignoreValue, else ref.current.value.
+ */
+export function getRefValue(ref, ignoreValue='', defaultValue=null) {
+  if (ref.current.value === ignoreValue) {
+    return defaultValue;
+  } 
+  return ref.current.value;
```

Comment:
This could be done in a ternary, but it's not required as this is completely fine. For more information: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator